### ISSUE TITLE
Which one is the real The Great Suspender?

### ISSUE THREAD

**Issue by longlongvip** [deanoemcke/thegreatsuspender#1335]
I find some The Great Suspenders in Chrome Web Store, but which one is true?
- [The Great Suspender Original](https://chrome.google.com/webstore/detail/the-great-suspender-origi/ahmkjjgdligadogjedmnogbpbcpofeeo)
- [The Great Suspender 1](https://chrome.google.com/webstore/detail/the-great-suspender/jaekigmcljkkalnicnjoafgfjoefkpeg)
- [The Great Suspender 2](https://chrome.google.com/webstore/detail/the-greater-suspender/ignechgpokdmbnbfpnhnjmhgdnlcopna)
- [The Marvellous Suspender](https://chrome.google.com/webstore/detail/the-marvellous-suspender/noogafoofpebimajpfpamcfhoaifemoa)

**Comment by wylie39** [comment]
NONE OF THEM. The original got taken down. I would recommend the marvelous suspender as a good alternative.

**Comment by longlongvip** [comment]
> NONE OF THEM. The original got taken down. I would recommend the marvelous suspender as a good alternative.

I get it

**Comment by baz357** [comment]
> I find some The Great Suspenders in Chrome Web Store, but which one is true?
> 
> * [The Great Suspender Original](https://chrome.google.com/webstore/detail/the-great-suspender-origi/ahmkjjgdligadogjedmnogbpbcpofeeo)
> * [The Great Suspender 1](https://chrome.google.com/webstore/detail/the-great-suspender/jaekigmcljkkalnicnjoafgfjoefkpeg)
> * [The Great Suspender 2](https://chrome.google.com/webstore/detail/the-greater-suspender/ignechgpokdmbnbfpnhnjmhgdnlcopna)
> * [The Marvellous Suspender](https://chrome.google.com/webstore/detail/the-marvellous-suspender/noogafoofpebimajpfpamcfhoaifemoa)

Yes, this is really confusing... all 3 The Great Suspender extensions have different version numbers (file sizes are also not the same). What a mess.

I'm currently using The Marvellous Suspender.

**Comment by mrizkimaulidan** [comment]
> NONE OF THEM. The original got taken down. I would recommend the marvelous suspender as a good alternative.

what happen with the original?

**Comment by kiratheone** [comment]
> > NONE OF THEM. The original got taken down. I would recommend the marvelous suspender as a good alternative.
> 
> what happen with the original?

You can check this issue #1263 @mrizkimaulidan

**Comment by aka3eka** [comment]
> > NONE OF THEM. The original got taken down. I would recommend the marvelous suspender as a good alternative.
> 
> what happen with the original?

https://www.ghacks.net/2021/01/09/we-no-longer-recommend-the-chrome-extension-the-great-suspender-here-is-why/

**Comment by varenc** [comment]
I would recommend this one: https://github.com/aciidic/thegreatsuspender-notrack

I like it because the code is easily readable and it does the same thing the old great suspender did.  Also I trust that the hive mind has audited the few changes to the code since the project was forked. You do need to jump through hoops to install a chrome extension from source that way, but the readme explains it. The extensions on the Chrome Store might be fine, and might even be the same code, but it's a bit more tedious to inspect the source code and look for changes. A bet at least one is malicious. Also with a Web Store extension the author can update the code remotely whenever they want.  By installing the source directly you'll know that the code never changes unless you change it.

( It's actually pretty ironic.  The extensions on the Chrome Web Store are dubious, but the non-Google audited extensions on Github are more trusted )

**Comment by longlongvip** [comment]
The issue should be closed, because it has more alternatives now
